packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -72,6 +169,271 @@ export function IntegrationList() {
+                            integrationMessages.motherDetails
+                          ),
+                          value: 'mothers-details'
+                        },
+                        {
+                          label: intl.formatMessage(
+                            integrationMessages.fatherDetails
+                          ),
+                          value: 'fathers-details'
+                        },
+                        {
+                          label: intl.formatMessage(
+                            integrationMessages.informantDetails
+                          ),
+                          value: 'informant-details'
+                        }
+                      ]}
+                      name="test-checkbox-group1"
+                      value={selectedItems}
+                      onChange={(newValue) => {
+                        setSelectedItems(newValue)
+                      }}
+                    />
+                    <Border></Border>
+
+                    {NoPII && (
+                      <CheckboxGroup
+                        id="test-checkbox-group1"
+                        options={[
+                          {
+                            label: intl.formatMessage(
+                              integrationMessages.registrationDetailsNoPII
+                            ),
+                            value: 'registration-details-noPII'
+                          },
+                          {
+                            label: intl.formatMessage(
+                              integrationMessages.childDetailsNoPII
+                            ),
+                            value: 'child-details-noPII'
+                          },
+                          {
+                            label: intl.formatMessage(
+                              integrationMessages.motherDetailsNoPII
+                            ),
+                            value: 'mothers-details-noPII'
+                          },
+                          {
+                            label: intl.formatMessage(
+                              integrationMessages.fatherDetailsNoPII
+                            ),
+                            value: 'fathers-details-noPII'
+                          },
+                          {
+                            label: intl.formatMessage(
+                              integrationMessages.informantDetailsNoPII
+                            ),
+                            value: 'informant-details-noPII'
+                          }
+                        ]}
+                        name="test-checkbox-group1"
+                        value={selectedItemsNoPII}
+                        onChange={(newValue) => setSelectedItemsNoPII(newValue)}
+                      />
+                    )}
+                    <DivSection
+                      style={{ display: 'flex', alignItems: 'flex-end' }}
+                    >
```

Comment:
[`<Stack>`](https://ui-kit.farajaland-qa.opencrvs.org/?path=/docs/layout-stack--default) can help you with simple flex tasks.